WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java

```diff
@@ -191,12 +195,28 @@ protected boolean isExpandableList() {
         private final List<FollowerModel> list;
         private final Activity context;
         private final LayoutInflater inflater;
+        private final  List<String> dotComUserBlogsURL = new ArrayList<>();
 
         public DotComFollowerAdapter(Activity context, List<FollowerModel> list) {
             super(context, R.layout.stats_list_cell, list);
             this.context = context;
             this.list = list;
             inflater = LayoutInflater.from(context);
+
+            // Read all the dotcomBlog blogs and get the list of home URLs.
+            // This will be used later to check if the user is a member of followers blog marked as private.
+            List <Map<String, Object>> dotComUserBlogs = WordPress.wpDB.getAccountsBy("dotcomFlag=1", new String[]{"homeURL"});
```

Comment:
Consider moving this off the UI thread since it has the potential to cause an ANR. That may sound unlikely, but I've seen some ANRs in the reader that were due to db lookups on the UI thread.